File: BancoMR/app/src/main/java/com/devheverton/bancomr/MainActivity.java
package com.devheverton.bancomr;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.devheverton.bancomr.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.imgSaldo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent navegarTelaDeSaldo = new Intent(MainActivity.this, Saldo.class);
                startActivity(navegarTelaDeSaldo);
finish();
            }
        });
        binding.imgFaturas.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent navegarTelasFaturas = new Intent(MainActivity.this, Faturas.class);
                startActivity(navegarTelasFaturas);
                finish();
            }
        });
        binding.imgTransferencia.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent navegarTelasTransferencia = new Intent(MainActivity.this, Transferencia.class);
                startActivity(navegarTelasTransferencia);
                finish();
            }
        });
        binding.imgPoupanA.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent navegarTelaPoupanca = new Intent(MainActivity.this, Poupanca.class);
                startActivity(navegarTelaPoupanca);
                finish();
            }
        });
    }
}

File: AppdeFilmes2/app/src/main/java/com/devheverton/appdefilmes/MainActivity.kt
package com.devheverton.appdefilmes

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper

import androidx.appcompat.app.AppCompatActivity

import com.devheverton.appdefilmes.view.FormLogin


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Handler(Looper.getMainLooper()).postDelayed({
           val intent = Intent(this, FormLogin::class.java)
            startActivity(intent)
            finish()

        },2000)
        }
    }
